arlo-client/src/components/CreateAudit.tsx

```diff
@@ -79,6 +79,11 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
       toast.error(err.message)
     }
   }
+
+  if (isAuthenticated === null) return null // Still loading
+  if (meta && meta.type === 'audit_board') {
+    return <div>404 Not Found</div>
```

Comment:
is this a fake 404? :)

Reply:
yeah, before this view (which is at the root path) was crashing if you were logged in as an audit board, so i just wanted to put in something slightly better